ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dhcp-relay.j2

```diff
@@ -15,4 +15,9 @@ dhcp relay
 {%     elif dhcp_relay.tunnel_requests_disabled is arista.avd.defined(false) %}
    no tunnel requests disabled
 {%     endif %}
+{%     if dhcp_relay.mlag_peerlink_requests_disabled is arista.avd.defined(true) %}
+   mlag peer-link requests disabled
+{%     elif dhcp_relay.mlag_peerlink_requests_disabled is arista.avd.defined(false) %}
+   no mlag peer-link requests disabled
+{%     endif %}
```

Comment:
is this required? We are not testing for it an looking at EOS CLI:


```
ceos2(config)#dhcp relay
ceos2(config-dhcp-relay)#mlag peer-link requests disabled
ceos2(config-dhcp-relay)#show active
dhcp relay
   mlag peer-link requests disabled
   server 10.0.0.1
ceos2(config-dhcp-relay)# no mlag peer-link requests disabled
ceos2(config-dhcp-relay)#show active
dhcp relay
   server 10.0.0.1
```

Reply:
I just did the same as for the already extisting option:

```
{%     if dhcp_relay.tunnel_requests_disabled is arista.avd.defined(true) %}
   tunnel requests disabled
{%     elif dhcp_relay.tunnel_requests_disabled is arista.avd.defined(false) %}
   no tunnel requests disabled
{%     endif %}
```

Reply:
I don't think there is any other command that could affect the default (unset) behavior, so we only need the non-default option (true in this case)

Reply:
Should we then also adjust the existing option `tunnel requests disabled`?

```
KHM-t-dehesw-spline-101__13:51:53>ena
KHM-t-dehesw-spline-101__13:51:54#conf t
KHM-t-dehesw-spline-101__13:51:59(config)#dhcp relay 
KHM-t-dehesw-spline-101__13:52:07(config-dhcp-relay)#tunnel requests disabled 
KHM-t-dehesw-spline-101__13:52:15(config-dhcp-relay)#show ac
dhcp relay
   tunnel requests disabled
KHM-t-dehesw-spline-101__13:52:18(config-dhcp-relay)#no tunnel requests disabled 
KHM-t-dehesw-spline-101__13:52:26(config-dhcp-relay)#
KHM-t-dehesw-spline-101__13:52:27(config-dhcp-relay)#show ac
dhcp relay
KHM-t-dehesw-spline-101__13:52:33(config-dhcp-relay)#
KHM-t-dehesw-spline-101__13:52:33(config-dhcp-relay)#show run all section dhcp relay
dhcp relay
   no tunnel requests disabled
   no mlag peer-link requests disabled
 ```

Reply:
Hi @kmueller68 - indeed it was there for not much either - I have cleaned it up in my commit I just pushed to your branch.
If the CI passes I will approve and merge. Thanks for the contribution!

Reply:
Thank you very much